src/ucs/async/async.h

```diff
@@ -118,6 +119,24 @@ static inline int ucs_async_is_blocked(const ucs_async_context_t *async)
     return async->poll_block > 0;
 }
 
+/**
+ * Returns wether or not thread ownership has changed
```

Comment:
typo: whether
I would remove "or not", but minor, up to you

Reply:
fixed